src/main/resources/BridgeServer2.conf

```diff
@@ -77,21 +85,11 @@ dev.synapse.tracking.view = syn20683692
 uat.synapse.tracking.view = syn20683693
 prod.synapse.tracking.view = syn11956745
 
-# AWS credentials for doing pre-signed upload
-aws.key.upload = dummy-value
-aws.secret.key.upload = dummy-value
-
 # Upload buckets
-local.upload.bucket = org-sagebridge-upload-local
-dev.upload.bucket = org-sagebridge-upload-develop
-uat.upload.bucket = org-sagebridge-upload-uat
-prod.upload.bucket = org-sagebridge-upload-prod
+local.upload.bucket = org-sagebridge-upload-${bucket.suffix}
```

Comment:
Should this be upload.bucket instead of local.upload.bucket? (Here and everywhere else.)